lib/jellyfish/config_reader.ex

```diff
@@ -128,20 +128,28 @@ defmodule Jellyfish.ConfigReader do
     end
   end
 
+  def read_components_used() do
```

Comment:
Have you consider injecting this from config file as a list of components instead of a set of true/false key/value? 

What I mean is something like in config.exs or somwewhere:

```elixir
config :jellyfish, 
  components: [
    Jellyfish.Component.SIP,
    Jellyfish.Component.Recording,
    ...
  ]
```

Reply:
In runtime.exs this could be passed as from env which could be a string which we later split and have a ready to go list and is easy to change.

Reply:
Great suggestion. Changed